block-node/messaging/build.gradle.kts

```
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+plugins { id("org.hiero.gradle.module.library") }
+
+description = "Hiero Block Node Messaging"
+
+// Remove the following line to enable all 'javac' lint checks that we have turned on by default
+// and then fix the reported issues.
+tasks.withType<JavaCompile>().configureEach { options.compilerArgs.add("-Xlint:-exports") }
+
+mainModuleInfo {
+    annotationProcessor("dagger.compiler")
+    annotationProcessor("com.google.auto.service.processor")
+    runtimeOnly("com.swirlds.config.impl")
+    runtimeOnly("org.apache.logging.log4j.slf4j2.impl")
```

should we link this with a todo?
```suggestion
    // @todo(871) remove as part of ticket
    runtimeOnly("org.apache.logging.log4j.slf4j2.impl")
```

---

just a nit,  feel free to resolve and merge.

---

I'll make sure it's handled as part of 871 if this merges first.